### PR TITLE
[BUGFIX] fix update data docs as validation action

### PR DESCRIPTION
**Pull request by alexsherstinsky**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
-
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [ ] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [ ] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [ ] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md [hunk 1]
```diff
@@ -21,51 +21,51 @@ Steps
...
-    2. Run the Checkpoint and verify that no errors are thrown. You can run the Checkpoint from the CLI as explained in [How to run a Checkpoint in terminal](../checkpoints/how-to-run-a-checkpoint-in-terminal) or from Python, as explained in [How to run a Checkpoint in python](../checkpoints/how-to-run-a-checkpoint-in-python).
-    3. Check your configured Data Docs sites to confirm that a new Validation Result has been added.
+   1. Open the configuration file of a Checkpoint you created earlier and replace the value of ``validation_operator_name`` with the name of the Validation Operator you added the ``UpdateDataDocs`` action to. The details of Checkpoint configuration can be found in this [How to add validations data or suites to a Checkpoint](../../../guides/validation/checkpoints/how-to-add-validations-data-or-suites-to-a-checkpoint).
+   2. Run the Checkpoint and verify that no errors are thrown. You can run the Checkpoint from the CLI as explained in [How to run a Checkpoint in terminal](../checkpoints/how-to-run-a-checkpoint-in-terminal) or from Python, as explained in [How to run a Checkpoint in python](../checkpoints/how-to-run-a-checkpoint-in-python).
```
Comment (cdkini) [inline docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md]: Links seem to be broken in Netlify build?

Comment (cdkini) [reply docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md]: Please capitalize Python 🐍

**Review by cdkini — APPROVED** [review]
Please see my comments but LGTM after!

File: docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md (hunk 1 above)
Comment (alexsherstinsky) [reply docs/guides/validation/validation_actions/how-to-update-data-docs-as-a-validation-action.md]: Done!  Nice catch (as it is capitalized in the linked document).

**Pull request merged**